core/modules/ccontrol/MergingRequester.cc

```diff
@@ -227,11 +227,10 @@ bool MergingRequester::_merge() {
 }
 
 void MergingRequester::_setError(int code, std::string const& msg) {
-    LOGF_INFO("setError");
-    LOGF_INFO("setError %1% %2%" % code % msg);
+    LOGF_INFO("setError: code: %1%, message: %2%" % code % msg);
     std::lock_guard<std::mutex> lock(_errorMutex);
-    _error.code = code;
-    _error.msg = msg;
+    _error.setCode(code);
+    _error.setMsg(msg);
```

Comment:
Reasonable compiler should be able to optimize that temporary away (if all methods are inline). Even if not the performance penalty should be negligible unless you need to create billions of Error instances.